tools/c_native/cnativenodelib.c

```diff
@@ -22,26 +22,34 @@ FILE* treeFp;
 int objectType = -1;  // declared at this level to propagate type from rbranch context to its element children contexts
 
 int stringToTypeDef(char* type) {
-    if (strcmp(type, "Int8") == 0)
+    if (strcmp(type, "Int8") == 0 || strcmp(type, "int8") == 0)
         return INT8;
-    if (strcmp(type, "UInt8") == 0)
+    if (strcmp(type, "UInt8") == 0 || strcmp(type, "uint8") == 0)
         return UINT8;
-    if (strcmp(type, "Int16") == 0)
+    if (strcmp(type, "Int16") == 0 || strcmp(type, "int16") == 0)
         return INT16;
-    if (strcmp(type, "UInt16") == 0)
+    if (strcmp(type, "UInt16") == 0 || strcmp(type, "uint16") == 0)
         return UINT16;
-    if (strcmp(type, "Int32") == 0)
+    if (strcmp(type, "Int32") == 0 || strcmp(type, "int32") == 0)
         return INT32;
-    if (strcmp(type, "UInt32") == 0)
+    if (strcmp(type, "UInt32") == 0 || strcmp(type, "uint32") == 0)
         return UINT32;
-    if (strcmp(type, "Double") == 0)
+    if (strcmp(type, "Double") == 0 || strcmp(type, "double") == 0)
         return DOUBLE;
-    if (strcmp(type, "Float") == 0)
+    if (strcmp(type, "Float") == 0 || strcmp(type, "float") == 0)
         return FLOAT;
-    if (strcmp(type, "Boolean") == 0)
+    if (strcmp(type, "Boolean") == 0 || strcmp(type, "boolean") == 0)
         return BOOLEAN;
-    if (strcmp(type, "String") == 0)
+    if (strcmp(type, "String") == 0 || strcmp(type, "string") == 0)
         return STRING;
+    if (strcmp(type, "sensor") == 0)
+        return SENSOR;
+    if (strcmp(type, "actuator") == 0)
+        return ACTUATOR;
+    if (strcmp(type, "streamsensor") == 0)
```

Comment:
[Here](https://github.com/GENIVI/vehicle_signal_specification#sensoractuator-data-entry) I put just ```stream``` to make it shorter and I thought it would be enough to understand. I don't care much which way to go. What's your opinion @UlfBj?